Common/src/Storage/TaskTableExtensions.cs

```diff
@@ -509,18 +522,20 @@ public static class TaskTableExtensions
   {
     var res = await taskTable.UpdateOneTask(taskData.TaskId,
                                             x => x.OwnerPodId == "" && x.Status == TaskStatus.Submitted,
-                                            new List<(Expression<Func<TaskData, object?>> selector, object? newValue)>
-                                            {
-                                              (tdm => tdm.OwnerPodId, taskData.OwnerPodId),
-                                              (tdm => tdm.OwnerPodName, taskData.OwnerPodName),
-                                              (tdm => tdm.ReceptionDate, taskData.ReceptionDate),
-                                              (tdm => tdm.AcquisitionDate, taskData.AcquisitionDate),
-                                              (tdm => tdm.Status, TaskStatus.Dispatched),
-                                            },
+                                            new UpdateDefinition<TaskData>().Set(tdm => tdm.OwnerPodId,
+                                                                                 taskData.OwnerPodId)
+                                                                            .Set(tdm => tdm.OwnerPodName,
+                                                                                 taskData.OwnerPodName)
+                                                                            .Set(tdm => tdm.ReceptionDate,
+                                                                                 taskData.ReceptionDate)
+                                                                            .Set(tdm => tdm.AcquisitionDate,
+                                                                                 taskData.AcquisitionDate)
+                                                                            .Set(tdm => tdm.Status,
+                                                                                 TaskStatus.Dispatched),
                                             cancellationToken: cancellationToken)
                              .ConfigureAwait(false);
 
-    taskTable.Logger.LogInformation("Acquire task {task} on {podName} with {success}",
+    taskTable.Logger.LogInformation("Acquire task {Task} on {PodName} with {Success}",
```

Comment:
why changing that ? It will change the filters on logs

Reply:
I think we should transform all logs to use capitals, but I agree it is not the right PR to do it.